source/scpp/build.d

```diff
@@ -29,26 +29,64 @@ immutable SourcePath  = RootPath;
 immutable BuildPath   = RootPath.buildPath("build");
 immutable BuildTarget = BuildPath.buildPath("libscp.o");
 
-/// Path passed to `-I`
+/// Path passed to `/I`
 immutable Includes = [
     SourcePath,
     SourcePath.buildPath("src"),
     SourcePath.buildPath("lib"),
     SourcePath.buildPath("lib", "xdrpp"),
+    SourcePath.buildPath("lib", "libsodium", "include"),
```

Comment:
Note that should shouldn't be necessary if `libsodium` is in the compiler search path